godaddy-email-marketing.php

```diff
@@ -231,9 +233,50 @@ public function deactivate() {
 	}
 
 	/**
+	 * Enqueue admin scripts
+	 *
+	 * @since 1.1.4
+	 */
+	public function register_admin_scripts() {
+
+		if ( get_option( 'wpem-gem-notice' ) ) {
+
+			$suffix = WP_DEBUG ? '' : '.min';
+
+			wp_enqueue_script( 'gem-notice', GEM_PLUGIN_URL . "js/gem-notice{$suffix}.js", array( 'jquery' ), GEM_VERSION );
+
+			wp_localize_script( 'gem-notice', 'gem_notice', [
+				'ajax_nonce' => wp_create_nonce( 'dismiss_gem_notice_nonce' ),
+			] );
+
+		}
+
+	}
+
+	/**
 	 * Displays the admin notice.
 	 */
 	public function action_admin_notices() {
+
+		if ( get_option( 'wpem-gem-notice' ) ) {
```

Comment:
@EvanHerman Let's add a private function to get rid of this `if`. `$this->display_gem_notice` or something that makes sense.